cmd/samples/pubsub/multisender/main.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/kelseyhightower/envconfig"
+
+	"github.com/cloudevents/sdk-go"
+	cecontext "github.com/cloudevents/sdk-go/pkg/cloudevents/context"
+	cloudeventspubsub "github.com/cloudevents/sdk-go/pkg/cloudevents/transport/pubsub"
```

Comment:
For consistency with other changes in this PR, this should be abbreviated "cepubsub".